code/script.js

```diff
@@ -1,47 +1,205 @@
 // All the DOM selectors stored as short variables
-const chat = document.getElementById('chat')
+const theWholeForm = document.getElementById('form-wrapper');
+const submitButton = document.getElementById('submit-ID') 
 
-// Global variables, if you need any, declared here
+//Global variables
+let questionNumber = 1; //stores the question number
+let TypeOfArrangement = ''; //stores type of arrangement to print in the order confirmation
 
-// Functions declared here
+//questionHandler function, passes the input from the user, 
+//plays sound effect and calls the functiond depending on the question number 
+const questionHandler = (userInput) => {
+  showMessage(userInput, 'user');
+  playSoundUser ()
+  switch (questionNumber) {
+    case 1: 
+      setTimeout(() => showFlowerArrangement(userInput), 1000); 
+      break;
+    case 2: 
+      setTimeout(() => showTypeOfArrangement(userInput), 1000);
+      break;
+    case 3: 
+      setTimeout(() => showFlowerSelector (userInput), 1500);
+      break;
+    case 4: 
+      setTimeout(() => showFinalConfirmation (userInput), 2000);
+      break;
+    default:
+      setTimeout(() => thankYou(), 1000);
+      break;
+  }
+};
 
-// This function will add a chat bubble in the correct place based on who the sender is
+//Function that shows messages
 const showMessage = (message, sender) => {
+  const chat = document.getElementById('chat');
   if (sender === 'user') {
     chat.innerHTML += `
       <section class="user-msg">
         <div class="bubble user-bubble">
-          <p>${message}</p>
+          <p class="message">${message}</p>
         </div>
-        <img src="assets/user.png" alt="User" />  
+        <img src="assets/flowerbotuser.png" alt="User" />  
       </section>
     `
   } else if (sender === 'bot') {
     chat.innerHTML += `
       <section class="bot-msg">
-        <img src="assets/bot.png" alt="Bot" />
+        <img src="assets/flowerbot.png" alt="Bot" />
         <div class="bubble bot-bubble">
-          <p>${message}</p>
+          <p class="message">${message}</p>
         </div>
       </section>
-    `
+     `
   }
-  // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
   chat.scrollTop = chat.scrollHeight
 }
 
-// Starts here
+//Greeting function
 const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+  showMessage(`Hello there, What's your name?`, 'bot');
+  
+}
+// Function that takes adds event listener to the form submission
+const initFncn = () => {
```

Comment:
Please avoid using this kind of shortcuts. It makes it a little bit harder to read. 🤓